components/combobox/index.css

```
@@ -377,6 +405,22 @@
 		&.is-invalid .spectrum-Textfield-validationIcon {
 			inset-inline-end: var(--mod-combobox-button-width, var(--spectrum-combobox-button-width));
 		}
+
+		&.spectrum-Textfield--quiet {
```

This is super close! As always, feel free to get a second opinion. I would probably do a similar approach to one I mentioned above, just without the "not quiet": 

```suggestion
&.is-readOnly {
	.spectrum-Combobox-input:read-only {
		border-block-end: var(--mod-combobox-border-width, var(--spectrum-combobox-border-width)) solid var(--mod-combobox-readonly-input-border-color, var(--spectrum-combobox-readonly-input-border-color));
	}
				
	&.is-disabled .spectrum-Combobox-input:read-only {
		color: var(--highcontrast-textfield-text-color-disabled, var(--mod-textfield-text-color-disabled, var(--spectrum-combobox-readonly-text-color-disabled)));
		border-color: var(--mod-textfield-border-color-disabled, var(--spectrum-combobox-border-color-disabled));
	}
			
	&.is-invalid > .spectrum-Combobox-input:read-only {
		border-block-end-color: var(--highcontrast-textfield-border-color-invalid-default, var(--mod-textfield-border-color-invalid-default, var(--spectrum-combobox-readonly-border-color-invalid-default)));
	}
}
```